openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -4,13 +4,13 @@
     {% for part in answer.parts %}
         <li class="submission__answer__part">
             <article class="submission__answer__part__prompt">
-                <h2 class="submission__answer__part__prompt__title">{% trans "The question for this section." %}</h2>
+                <h3 class="submission__answer__part__prompt__title">{% trans "The question for this section." %}</h3>
                 <div class="submission__answer__part__prompt__value">
                     {{ part.prompt.description|linebreaks }}
                 </div>
             </article>
             <div class="submission__answer__part__text">
-                <h2 class="submission__answer__part__text__title">{{ answer_text_label }}</h2>
+                <h3 class="submission__answer__part__text__title">{{ answer_text_label }}</h3>
```

Comment:
I kept running into an overriding style coming from https://github.com/edx/edx-platform/blob/master/lms/static/sass/course/courseware/_courseware.scss#L453 for the `h2` element here.

Is changing it to an `h3` an acceptable workaround? Is there some other way to say "don't apply that style in this case" in CSS?